Content/Tools/Gameplay/FastForward.cs

```diff
@@ -12,19 +12,13 @@ internal class FastForward : Tool
 
 		public override string IconKey => "FastForward";
 
-		public override string DisplayName => "Fast forward";
-
-		public override string Description => "Click to speed up the game, up to 4x. Right click to move backwards through speeds.";
-
 		public override bool HasRightClick => true;
 
-		public override string RightClickName => "Decrease fast forward rate";
-
 		public override void OnActivate()
 		{
 			if (Main.netMode != NetmodeID.SinglePlayer)
 			{
-				Main.NewText("Fast forward is disabled in multiplayer", Color.Red);
+				Main.NewText(LocalizationHelper.GetToolText("Difficulty.MultiplayerDisabled"), Color.Red);

```

Comment:
Is it correct to re-use the key from the difficulty tool here? Potential oversight

If we want to minimize keys perhaps just a 'This is disabled in multiplayer' key used across all tools would probably be fine